doc/glossary.rst

```diff
@@ -126,10 +126,13 @@ reflect either technical terms or legacy ways of referring to Ceph systems.
 
 	Ceph Manager Dashboard
 	Ceph Dashboard
+	Dashboard Module
 	Dashboard Plugin
 	Dashboard
-                A built-in web based monitoring and administration application to
-                Ceph Manager. Refer :ref:`mgr-dashboard` for more details.
+        A built-in web-based Ceph management and monitoring application to
```

Comment:
I think the indention is incorrect here. Can be easily checked when pressing the 'View file' button.

Reply:
Thanks for your comment. I modified the indentation intentionally, so it's aligned with the other sections.